module_4/framework/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM php:8.1-cli
```

Comment:
@Hlopec This is not the best idea to use CLI here. How is this app supposed to deal with the higher load? What is the difference between PHP in CLI and PHP-FPM?

This solution may exist as a demo but it's not functional for a real web-server scenario.